min_tilt_version/Tiltfile

```diff
@@ -0,0 +1,24 @@
+def min_tilt_version(min_version):
+  # this assumes that the tit version command always returns
+  # v0.13.0, built 2020-04-01
```

Comment:
note that in dev mode, Tilt returns a string that looks like:
v0.13.0-dev, built 2020-04-03

Reply:
I will add a case for this too.